packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -178,6 +190,16 @@ function FrameElement({ id }) {
   );
   const handleMediaClick = useDoubleClick(NOOP, handleMediaDoubleClick);
 
+  /**
+   * Announce keyboard options on element.
+   *
+   * Using a live region because an `aria-label` would remove
+   * any labels/content that would be read from children.
+   */
+  const handleFocus = useCallback(() => {
+    speak(FRAME_ELEMENT_MESSAGE);
+  }, [speak]);
```

Comment:
Ahh thanks for doing this. I should reference this and do the same in the floating menu PR 🙌